Backpack/BarChart/Classes/MiniFlareView.swift

```diff
@@ -0,0 +1,68 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+
+@objcMembers
+@objc(BPKMiniFlareView)
+public final class BPKMiniFlareView: BPKFlareView {
+
+    let label: UILabel = BPKLabel()
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.setupViews()
+    }
+
+    public required init(coder: NSCoder) {
+        super.init(coder: coder)
+        self.setupViews()
+    }
+
+    func setupViews() {
+        self.backgroundColor = BPKColor.dynamicColor(withLightVariant: BPKColor.skyBlueShade02,
+                                                     darkVariant: BPKColor.blackTint06)
+        self.label.textColor = BPKColor.dynamicColor(withLightVariant: BPKColor.white, darkVariant: BPKColor.black)
+
+        self.contentView.addSubview(self.label)
+
+        self.label.translatesAutoresizingMaskIntoConstraints = false
+
+        self.cornerRadius = BPKSpacingSm
+
+        NSLayoutConstraint.activate([
+            self.contentView.widthAnchor.constraint(equalTo: self.label.widthAnchor, constant: 2*BPKSpacingMd),
+            self.contentView.heightAnchor.constraint(equalTo: self.label.heightAnchor, constant: 2*BPKSpacingMd),
+            self.label.centerXAnchor.constraint(equalTo: self.contentView.centerXAnchor),
+            self.label.centerYAnchor.constraint(equalTo: self.contentView.centerYAnchor)
+        ])
+    }
+
+    public var text: String? {
+        get {
+            return self.label.text
+        }
+        set {
+            self.label.text = newValue
+        }
+    }
+
+    override public func flareHeight() -> CGFloat {
```

Comment:
Note: I feel like this should be a readonly property in `BPKFlareView` and the same here. In Objective-C there is no distinction because a readonly property is just a method but I don't think using a method allows us to override it with a readonly property in Swift.

Reply:
So in FlareView.h it would be
```
@property(readonly, nonatomic) CGFloat flareHeight;
```

And FlareView.m would be unchanged
```
- (CGFloat)flareHeight {
    return BPKFlareHeight;
}
```

Then in MiniFlareView.swift we'd do
```
    override public var flareHeight: CGFloat {
        get {
            return BPKSpacingMd
        }
        set {
            return
        }
    }
```

Reply:
Yeah except in Swift it just needs to be

```swift
override public var flareHeight: CGFloat {
    BPKSpacingMd
}
```

Because of implicit returns in getters and the ability to omit `get` when implemnting a read-only property